mmv1/third_party/terraform/go.mod.erb

```diff
@@ -3,33 +3,36 @@ module github.com/hashicorp/terraform-provider-google
 
 go 1.21
 
+toolchain go1.21.4
```

Comment:
This is a new feature to Go 1.21 : https://go.dev/doc/toolchain

It was added automatically, I think because I modified the Go version in go.mod via CLI instead of direct edit.

I'll remove this and we can add toolchain as a deliberate choice later

Reply:
[023d65c](https://github.com/GoogleCloudPlatform/magic-modules/pull/10109/commits/023d65cb52324d1be4e1f7ddc98d9151999f5f1a)